sandbox/src/main/scala/example/Sandbox.scala

```diff
@@ -17,7 +17,14 @@ import scala.scalajs.js.annotation.*
 import scalajs.js
 
 @JSExportTopLevel("TyrianApp")
-object Sandbox extends TyrianApp[Msg, Model]:
+object Sandbox extends MultiPage[Msg, Model]:
+
+  val hashResultToMessage: Navigation.Result => Msg = {
+    case Navigation.Result.CurrentHash(hash) => Msg.NavigateTo(Page.fromString(hash))
+    case _                                   => Msg.NavigateTo(Page.Page1)
```

Comment:
🤔 wondering if there's a way to bake those `NavigateTo` messages into `Msg` generic type somehow. They look like they'll be always used when dealing with a `MultiPage`, so why not enforce/facilitate them? Probably not a topic for this PR but the next one generalizing the router.

Reply:
So I think at the moment I'm not going to worry about this. The router requires you to decide what to do when a route changes by producing a `Msg`. As you say, maybe follow up in another PR if there turns out to be a common solution, but I don't want to second guess one at the moment.